integration/src/main/java/org/apache/iotdb/integration/env/ClusterEnvBase.java

```diff
@@ -124,17 +123,8 @@ public void testWorking() throws InterruptedException {
 
       try (IoTDBConnection connection = getConnection(60);
           Statement statement = connection.createStatement()) {
-        statement.execute("SET STORAGE GROUP TO root.test" + counter);
-        statement.execute(
-            "CREATE TIMESERIES root.test" + counter + ".d0.s0 WITH DATATYPE=INT32, ENCODING=RLE");
-        if (statement.execute("SHOW TIMESERIES")) {
-          ResultSet resultSet = statement.getResultSet();
-          if (resultSet.next()) {
-            statement.execute("DELETE STORAGE GROUP root.*");
-            break;
-          }
-        }
-
+        statement.execute("SHOW TIMESERIES");
```

Comment:
The statement "SHOW TIMESERIES" may return an empty result when the cluster is not already.

Reply:
In this PR https://github.com/apache/iotdb/pull/4388 , any error responsed by other nodes will be handled and returned client with a MetaDataException.